### PR TITLE
BEGONE, CULTMAN

### PR DESCRIPTION
**Pull request by Geevies**
* Nerfs stun talisman.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -1058,15 +1058,14 @@ var/list/sacrificed = list()
 				O.show_message(text("<span class='warning'><B>[] invokes a talisman at []</B></span>", user, T), 1)
 
 			if(issilicon(T))
-				T.Weaken(15)
+				flick("e_flash", T.flash)
+				T.silent += 15
 				admin_attack_log(user, T, "Used a stun rune.", "Was victim of a stun rune.", "used a stun rune on")
```
Comment (MarinaGryphon) [inline code/game/gamemodes/cult/runes.dm]: Says stun rune even though stun rune and stun talisman are different now.

File: code/game/gamemodes/cult/runes.dm [hunk 2]
```diff
@@ -1058,15 +1058,14 @@ var/list/sacrificed = list()
...
 			else if(iscarbon(T))
 				var/mob/living/carbon/C = T
 				flick("e_flash", C.flash)
-				if (!(HULK in C.mutations))
+				if(!(HULK in C.mutations))
 					C.silent += 15
-				C.Weaken(25)
-				C.Stun(25)
 				admin_attack_log(user, C, "Used a stun rune.", "Was victim of a stun rune.", "used a stun rune on")
```
Comment (MarinaGryphon) [inline code/game/gamemodes/cult/runes.dm]: Says stun rune even though stun rune and stun talisman are different now.

File: code/game/gamemodes/cult/runes.dm (hunk 1 above)
Comment (Geevies) [inline code/game/gamemodes/cult/runes.dm]:
```suggestion
				admin_attack_log(user, T, "Used a stun talisman.", "Was victim of a stun talisman.", "used a stun talisman on")
```

File: code/game/gamemodes/cult/runes.dm (hunk 2 above)
Comment (Geevies) [inline code/game/gamemodes/cult/runes.dm]:
```suggestion
				admin_attack_log(user, C, "Used a stun talisman.", "Was victim of a stun talisman.", "used a stun talisman on")
```

**Comment by Kaedwuff** [comment]
I think they still need to offer a short-term stun, at the very least long enough to make people drop what they're holding but not be rendered helpless on the floor for a geological epoch. Like 2 to 3 seconds.

Otherwise why is it even called the stun talisman.

**Comment by Kaedwuff** [comment]
I think they still need to offer a short-term stun, at the very least long enough to make people drop what they're holding but not be rendered helpless on the floor for a geological epoch. Like 2 to 3 seconds.

Otherwise why is it even called the stun talisman. As this is now you might as well pick up a regular flash, because it's exactly as effective only has multiple uses.

Also, you have done nothing to update in-game information about this. You can't just change a game feature half-assedly and leave the cult guide with out of date information on it.

If you want to keep what you've done here, you need to also update the name of the talisman/rune to not be 'stun', and also redo the on-ground rune to not be horse shit and have a useful purpose, because right now you're almost better off not making a 'stun talisman' and just trying flash everyone in the area with the rune version, or just using the AOE blind rune instead.

**Comment by Geevies** [comment]
`Otherwise why is it even called the stun talisman. As this is now you might as well pick up a regular flash, because it's exactly as effective only has multiple uses.`
This goes through flash protection.

`Also, you have done nothing to update in-game information about this. You can't just change a game feature half-assedly and leave the cult guide with out of date information on it.`
1) Which guide?
2) Half-assedly?

`Right now you're almost better off not making a 'stun talisman' and just trying flash everyone in the area with the rune version, or just using the AOE blind rune instead.`
Sounds good.

`The rune version was intentionally bad because the talisman was very good.`
Sounds like something that should be fixed.

`Also, if this is in regards to the suggestion thread recently posted, you should connect the two somehow.`
The PR link was posted in the suggestion thread here: https://forums.aurorastation.org/topic/13093-nerf-the-stun-talisman/

**Comment by Kaedwuff** [comment]
iWKad22

-The cult tome has a guide which indicates the purposes and effects of all the runes. You have not updated it.  Two months down the line, new players will not see this changelog immediately visible, and they will be reliant on what information is provided to them in-game.  This will lead to confusion and frustration, which is not a good design.
-This PR is not finished. You changed exactly one thing without following up and modifying all aspects of the game tied to it.
-No, it does _not_ sound good. When, post nerf, an upgraded, more difficult to obtain version of something is less useful than the basic version, you have created a bad nerf and need to reconsider your balance scaling choices.  You hating something is not really justification to make it so bad it's not worth getting.
-Yes.  By you, since you are the one changing the feature.  Or rather, half the feature, then deciding the rest isn't your problem, apparently.
-Generally helpful to link suggestion threads in the actual PR description when you make them, wich is why I pointed it out.

**Comment by Geevies** [comment]
Updated the tome descriptions. Thanks for letting me know in the nicest tone you could muster.

As for the rest of your concerns. I don't care. Also, we're doing something cool behind the curtain. Maybe. Hopefully. We'll see.

**Comment by Kaedwuff** [comment]
You're right. I'm being too confrontational and losing my temper over a dumb thing on the internet again.

You have my apologies, and my gratitude for fixing at least one of the things I mentioned.

I'll probably come back at a later time and fix the rest.

**Comment by Geevies** [comment]
All good man, apology accepted. If you want me to look at something that concerns you, feel absolutely free to ding me at Geeves#8863 on discord.

**Pull request merged**